lib/zlib.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+require_relative "zlib/zlib"
```

Comment:
`require_relative` doesn't work because shared object files are not placed in directories for scripts.

Reply:
Fixed. (I guess this applies to the trunk repository / bundled gems?)